simtools/camera_efficiency.py

```diff
@@ -469,12 +508,20 @@ def calc_reflectivity(self):
             [wl_now > 299 and wl_now < 551 for wl_now in self._results["wl"]]
         ]
         c2_sum = np.sum(c2_reduced_wl)
+        cher_spec_weighted_reflectivity = c2_sum / c1_sum / self._results["masts"][0]
```

Comment:
I only assigned it in order to mention it in the docstring, otherwise it would look ugly (see below), hence, I suggest to keep the defined variable, or any other idea?
"""
        """
        Calculate the Cherenkov spectrum weighted reflectivity in the range 300-550 nm.

        Returns
        -------
        c2_sum / c1_sum / self._results["masts"][0]
            Cherenkov spectrum weighted reflectivity (300-550 nm)
        """
"""

Reply:
Hmm, what does it add if you put a variable name into the docstring? What only setting:

```
Returns
--------
Float
   Cherenkov spectrum weighted reflectivity (300-550 nm)
```

The name of the parameter is not required, see https://numpydoc.readthedocs.io/en/latest/format.html#returns

Reply:
Ok, then I will pass only the name, but the consistency will be lost since in all the other functions the variable is returned

Reply:
But you agree that this is clearer and simpler? We should rather apply in future this to the other methods.

Reply:
Indeed, it is not necessary to know the name of the returned variable since one has to define anyways a new one to save the returned value